datamade/ccfp-asset-dashboard
# Added the Priority Scoring UI + Form

## Overview
See PR title. This closes issue #18.

### Demo
![priority-scoring](https://user-images.githubusercontent.com/38969506/104056954-e9013400-51b6-11eb-8541-5a7ebce90c5a.gif)

### Notes
This PR doesn't have full styling.

## Testing Instructions
- visit deployment https://ccfp-asset-d-priority-s-gc0kvb.herokuapp.com/projects/
- add a project
- make sure you can still update the project "overview" form
- update the priority score
- test the form validation. those fields can only be a number between 1 and 5.
- double check that the UI reflects your saved changes
